Content.Server/CombatMode/CombatModeSystem.cs

```diff
@@ -39,6 +41,18 @@ private void OnEntityActionPerform(EntityUid uid, SharedCombatModeComponent comp
                 return;
 
             var attemptEvent = new DisarmAttemptEvent(args.Target, args.Performer);
+
+            if (EntityManager.TryGetComponent<HandsComponent>(args.Target, out HandsComponent targetHandsComponent)
+                && targetHandsComponent.ActiveHand != null
+                && !targetHandsComponent.ActiveHand.IsEmpty)
+            {
+                Debug.Assert(targetHandsComponent.ActiveHand.HeldEntity != null, "targetHandsComponent.ActiveHand.HeldEntity != null");
```

Comment:
wat, just remove this

Reply:
If I don't do an assert , Rider will be mad that held entity could be null

Reply:
Oh nvm I fixed it